core/src/Kokkos_Core_fwd.hpp

```diff
@@ -173,7 +173,7 @@ typedef Serial DefaultExecutionSpace;
 #endif
 
 #if defined(KOKKOS_ENABLE_DEFAULT_DEVICE_TYPE_OPENMP)
-typedef OpenMP DefaultHostExecutionSpace;
+using DefaultHostExecutionSpace = OpenMP;
 #elif defined(KOKKOS_ENABLE_DEFAULT_DEVICE_TYPE_THREADS)
 typedef Threads DefaultHostExecutionSpace;
```

Comment:
For consistency we should change all or none of these.  It actually matters for the clang-tidy work since `typedef` and `using` are not the same from an AST perspective.